django_keycloak_auth/settings.py

```diff
@@ -155,7 +155,7 @@
     'KEYCLOAK_SERVER_URL': 'http://localhost:8080/auth',
     'KEYCLOAK_REALM': 'TEST',
     'KEYCLOAK_CLIENT_ID': 'client-backend',
-    'KEYCLOAK_CLIENT_SECRET_KEY': 'E2n41fJgl9BPIS3nDk1DQQ7BIPf6PauH',
+    'KEYCLOAK_CLIENT_SECRET_KEY': '',
     'KEYCLOAK_CACHE_TTL': 60,
     'LOCAL_DECODE': False,
```

Comment:
I believe the default case should be `True`, because you dont need to bug the auth server with every request and it will add a delay in the request lifecycle

Reply:
Thanks! I agree, but wanted to make sure the author gets to make the decision.

Reply:
I agree too!